### PR TITLE
Create .env with `nile init` and remove example.env

### LINKED ISSUE #244: remove example.env

### LINKED ISSUE #249: Create .env with `nile init`
It might be helpful to insert a comment illustrating how users should populate the .env i.e. `# PRIVATE_KEY_1 = 1234...`

### PR DESCRIPTION
**Pull request by andrew-fleming**
Resolves #249 and resolves #244.

### CODE REVIEW

File: src/nile/core/init.py [hunk 1]
```diff
@@ -14,6 +14,11 @@ def init():
     with open("accounts.json", "w") as file:
         file.write("{}")
 
+    logging.info("🔑  Creating .env file")
+
+    with open(".env", "w") as file:
+        file.write("# ALIAS = PRIVATE_KEY")
```
Comment (martriay) [inline src/nile/core/init.py:20]: what about adding it to the sample project directory?

**Review by martriay — COMMENTED** [review]
Good improvement :) . Left just a comment

File: src/nile/core/init.py (hunk 1 above)
Comment (andrew-fleming) [reply src/nile/core/init.py]: Good idea!

File: src/nile/base_project/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+# ALIAS = PRIVATE_KEY
```
Comment (martriay) [inline src/nile/base_project/.env:1]: let's add an actual private key, and a working file

Comment (andrew-fleming) [reply src/nile/base_project/.env:1]: Should we keep it commented like: `# PRIVATE_KEY = 0xe3e70682c2094cac629f6fbed82c07cd`?

And what do you mean by a working file?

Comment (martriay) [reply src/nile/base_project/.env:1]: i mean that it's ready to be used. why commenting the key?

Comment (andrew-fleming) [reply src/nile/base_project/.env:1]: Ohhhh my mistake, I understand. So let's not comment the key :)

Comment (martriay) [reply src/nile/base_project/.env:1]: are `0x` keys currently supported?

Comment (andrew-fleming) [reply src/nile/base_project/.env:1]: `0x` keys are indeed supported

**Review by ericnordelo — APPROVED** [review]
LGTM!

**Pull request merged**